authors/apps/authentication/serializers.py

```diff
@@ -141,3 +251,41 @@ def update(self, instance, validated_data):
         instance.save()
 
         return instance
+
+
+class ResetSerializer(serializers.Serializer):
+    email = serializers.EmailField()
+    token = serializers.CharField(max_length=255, read_only=True)
+
+    # List the fields that should b
+
+    def validate(self, data):
+        """
+        Validate the email enterd by the user that requests to reset account passaword
+        """
+        email = data.get('email', None)
+
+        # Check if user just posted an empty object.
+        # If none exists ask tell the user that the field is required
```

Comment:
@Darius-Ndubi, please re-word to clarify what's meant by "ask tell the user" here.